cms/templates/settings.html

```diff
@@ -94,10 +96,10 @@ <h3 class="title">${_("Course Summary Page")} <span class="tip">${_("(for studen
               <li class="action-item">
                 <%
                   email_subject = urllib.quote(_("Enroll in {course_display_name}").format(
-                      course_display_name = context_course.display_name_with_default_escaped
+                      course_display_name = context_course.display_name_with_default
                   ).encode("utf-8"))
                   email_body = urllib.quote(_('The course "{course_display_name}", provided by {platform_name}, is open for enrollment. Please navigate to this course at {link_for_about_page} to enroll.').format(
-                      course_display_name = context_course.display_name_with_default_escaped,
+                      course_display_name = context_course.display_name_with_default,
                       platform_name = settings.PLATFORM_NAME,
                       link_for_about_page = link_for_about_page
```

Comment:
@mushtaqak @robrap and I looked at this together, and we think it is correct as-is. The email_subject and email_body are properly url-encoded because they are used in the mailto link below.